make/photon/db/Dockerfile

```diff
@@ -18,15 +18,16 @@ RUN tdnf erase -y toybox && tdnf install -y util-linux net-tools
 
 VOLUME /var/lib/postgresql/data
 
-ADD ./make/photon/db/docker-entrypoint.sh /entrypoint.sh
-ADD ./make/photon/db/docker-healthcheck.sh /docker-healthcheck.sh
-RUN chmod u+x /entrypoint.sh /docker-healthcheck.sh
-ENTRYPOINT ["/entrypoint.sh"]
-HEALTHCHECK CMD ["/docker-healthcheck.sh"]
-
+COPY ./make/photon/db/docker-entrypoint.sh /entrypoint.sh
```

Comment:
Appreciate it's not your change that introduced this but this docker file is inconsistent with the others. It is copying to a different filename to where it was copied from.  E.g `docker-entrypoint.sh` becomes `entrypoint` 🤷‍♂️